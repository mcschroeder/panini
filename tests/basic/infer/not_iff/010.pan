not : (x:bool) -> {y:bool|y = true <=> x = false}

f10 = not true

f11 = not false

