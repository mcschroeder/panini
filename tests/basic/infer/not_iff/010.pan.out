f10 : {y:𝔹 | y = false}
f11 : {y:𝔹 | y = true}